components/infobox/wikis/formula1/infobox_league_custom.lua

```diff
@@ -46,7 +47,8 @@ function CustomInjector:parse(id, widgets)
 			Cell{name = 'Number of Teams', content = {args.team_number}},
 			Cell{name = 'Engine', content = {args.engine}},
 			Cell{name = 'Tyres', content = {args.tyres}},
-			Cell{name = 'Chassis', content = {args.chassis}}
+			Cell{name = 'Chassis', content = {args.chassis}},
+			Cell{name = 'Race Leaders', content = {args.leaders}}
```

Comment:
Out of interest, what is the usecase for this?
Is it information visible on the page already?

Reply:
Is it possible for the list of leaders to be rather long? Might be better to have it somewhere else on the page instead of in the Infobox.
I'd also suggest to have some standardized input (e.g. comma-separated) to have the potential to store this information in the future or enhance the display (e.g. with flags)

Reply:
1: If you think it does not take too much space in the infobox, okay for me.
3: You can use fontawesome v5 icons which should have all of these. Does not require the design team :)
4: Sure, you could do something like `|leader1=HAM|leader1laps=5` or something.